src/kafka-util/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "kafka-util"
+version = "0.1.0"
+authors = ["Brennan Vincent <brennan@materialize.io>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
nit: ditch this annoying boilerplate comment, if you go back through CI for something.